Dockerfile.builder

```diff
@@ -28,11 +28,11 @@ RUN <<EOT
     set -e
     apt-get update
     apt-get -y upgrade
-    buildDeps="build-essential busybox ca-certificates curl git gosu libbz2-dev libffi-dev libjpeg62-turbo-dev libmagic1 libldap2-dev libopenjp2-7-dev libpcre3-dev libpq-dev libsasl2-dev libssl-dev libtiff5-dev libxml2-dev libxslt1-dev wget zlib1g-dev"
+    buildDeps="build-essential busybox ca-certificates curl git gosu libbz2-dev libffi-dev libjpeg62-turbo-dev libmagic1 libsasl2-dev libldap2-dev libopenjp2-7-dev libpcre3-dev libpq-dev libsasl2-dev libssl-dev libtiff6 libtiff5-dev libxml2-dev libxslt1-dev wget zlib1g-dev"
```

Comment:
libsasl2-dev is now here twice. And, we need both libtiff6 and libtiff5-dev?

Reply:
The duplication is an error, I'll fix it.  And yes, apparently we need both in bookworm. libtiff5 doesn't exist anymore in the distro, libtiff5-dev still is and  works with libtiff6 as a library, it is mentioned it is a transitional package.

Reply:
The duplication of libsasl2-dev is still here

Reply:
Just removed